### PR TITLE
Fixes #21106 - gateway-less support for host images

### PR DESCRIPTION
**Pull request by lzap**
Gateway is not always needed.

### CODE REVIEW

File: app/views/foreman_bootdisk/host.erb [hunk 1]
```diff
@@ -31,7 +30,9 @@ echo Configuring net${idx} for static IP address <%= interface.ip %>
 ifopen net${idx}
 set netX/ip <%= interface.ip %>
 set netX/netmask <%= interface.subnet.mask %>
+<% if interface.subnet.gateway %>
```
Comment (timogoebel) [inline app/views/foreman_bootdisk/host.erb]: `.present?` to also catch an empty value?

**Comment by timogoebel** [comment]
@lzap: Moved the Redmine ticket to Bootdisk. When you fix my other comment, this is ready.

File: app/views/foreman_bootdisk/host.erb (hunk 1 above)
Comment (lzap) [reply app/views/foreman_bootdisk/host.erb]: Done!

**Comment by timogoebel** [comment]
@domcleal: Anything to add?

**Comment by ares** [comment]
@timogoebel any chance we could get this in? it seems it was waiting only on pressing the button :-)

**Comment by timogoebel** [comment]
Merged, thanks @lzap and @ares.

**Comment by ares** [comment]
Awesome, thanks!

**Pull request merged**